l1-contracts-foundry/script/DeployErc20.s.sol

```diff
@@ -77,7 +77,7 @@ contract DeployErc20Script is Script {
                 mint: token.mint
             });
             console.log("Token deployed at:", tokenAddress);
-            token.addr = tokenAddress;
+            config.tokens[i].addr = tokenAddress;
```

Comment:
Yes, because in this case, the ```token``` variable is just a copy and does not overwrite the config value. As a result, all token addresses were the default value (address(0)), which was visible in the deployment output configuration.